src/rpm/headers/lead.rs

```diff
@@ -94,17 +94,16 @@ impl Lead {
         let mut name_arr: [u8; 66] = [0; 66];
         name_arr.copy_from_slice(name);
 
-        //save unwrap here since we've checked length of slices.
         Ok(Lead {
-            magic: magic.try_into().unwrap(),
+            magic: RPM_MAGIC,
```

Comment:
We've already checked for equality with the constant value, so why not just use the constant value.